magento/pwa-studio
# Set fixed min-height on main page

## Description

The `Main` component, which renders the header, page, and footer, sets `min-height: 100vh` on the page. As intended, this style prevents the footer from ever appearing above the fold. Unfortunately, setting a height with relative units causes issues when the page is rendered in an `<iframe>`, so we'll have to remove it.

This PR changes to `min-height: 40rem` (`640px`) and addresses any fallout from that change.

## Related Issue
PWA-1243.

## Acceptance

### Verification Stakeholders
- @sirugh 
- @dpatil-magento

### Specification

### Verification Steps
1. Visit the home page.
2. Verify that the footer is at least `640px` below the header.
3. Repeat for `/cart`, `/checkout`, and other pages.
4. On an empty page or 404 page, ensure the footer is only `640px` below the header.

## Screenshots / Screen Captures (if appropriate)
<img width="612" alt="Cart page on mobile" src="https://user-images.githubusercontent.com/2110986/104386912-45d05780-54f3-11eb-9e3a-bb22ab8044f2.png">

## Checklist
* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
